File: BankApi/Model/Cliente.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Model
{
    public class Cliente : Persona
    {
        [Required(ErrorMessage = "Este campo es obligatorio")]
        public string Contraseña { get; set; } = string.Empty;

        [Required(ErrorMessage = "Este campo es obligatorio")]
        public EstadoCliente Estado { get; set; }
        public ICollection<Cuenta> Cuentas { get; set; } = new List<Cuenta>();
    }

    public enum EstadoCliente
    {
        Activo = 0,
        Inactivo = 1
    }
}

File: bank-web-app/src/app/interfaces/cliente.ts
import { Cuenta } from "./cuenta";

interface Persona{
    id: string;
    nombre: string;
    genero: string;
    edad: number
    identificacion: string;
    direccion: string;
    telefono: number;
}

export interface Cliente extends Persona {
    contraseña: string;
    estado: number; 
    cuentas: Cuenta[]
}
